### DISCUSSION TITLE
Table actions based on custom methods in policy

### DISCUSSION THREAD

**Discussion by bilogic** [filamentphp/filament#8839] [Help]
https://github.com/filamentphp/filament/blob/808cfda32476790257ed1adbc98ce46fd1f07444/packages/panels/src/Resources/Resource.php#L162-L195

Looking through the filament code, there are some generic policy handling methods such as `can(...)` and `authorize(...)`. 

Can we reuse these to introduce our own actions which can be controlled by a custom method in our policy file? e.g. add `function cancel(...)` to policy to control if the `cancel` action is shown or not.

**Comment by bilogic** [comment]
Nevermind, I found another way out

**Reply by pajkho** [reply]
How did you solve this? I am also looking for a way to use custom policy methods.
